pkg/liqoctl/status/info_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Info", func() {
+	var (
+		infoNode InfoNode
+	)
+	Context("Creating a new root InfoNode", func() {
+		JustBeforeEach(func() {
+			infoNode = newRootInfoNode("root")
+		})
+		It("should return a valid InfoNode", func() {
+			infoNodeTest := InfoNode{
+				title:     "root",
+				detail:    "",
+				nextNodes: nil,
+				data:      nil,
+			}
+			Expect(infoNode).To(Equal(infoNodeTest))
+		})
+	})
+	Context("Populating a root node with some sections", func() {
+		JustBeforeEach(func() {
+			infoNode = newRootInfoNode("root")
+			infoNode.addSectionToNode("section1", "detail1").
+				addSectionToNode("section1-1", "detail1-1")
+			infoNode.addSectionToNode("section2", "detail2").
+				addSectionToNode("section2-1", "detail2-1")
+		})
+		It("should return a valid InfoNode tree", func() {
+			Expect(infoNode.title).To(Equal("root"))
+			Expect(infoNode.nextNodes[0].title).To(Equal("section1"))
```

Comment:
Should check it has len 2, to prevent panics and return better error messages (same below).